.packit.yaml

```diff
@@ -0,0 +1,18 @@
+files_to_sync:
+  - dnf5.spec
+
+jobs:
+  - job: propose_downstream
+    trigger: release
+    dist_git_branches:
+        - fedora-rawhide
+        - fedora-38
+  - job: koji_build
+    trigger: commit
```

Comment:
Is it like after each commit the build is triggered?

Reply:
as I understood it, after a commit in dist-git, yes
https://packit.dev/docs/fedora-releases-guide/#koji-build-job

Reply:
I am just wondering, because compared to our `ci.yml` is seems like those jobs there are independent and I was rather thinking about this process being in a single pipeline.

Reply:
Oh, right. I was a bit confused if it is triggered by a commit in the dnf5 repo...